Doc/library/graphlib.rst

```diff
@@ -19,20 +19,20 @@
 
    Provides functionality to topologically sort a graph of hashable nodes.
 
-   A topological order is a linear ordering of the vertices in a graph such that
-   for every directed edge u -> v from vertex u to vertex v, vertex u comes
-   before vertex v in the ordering. For instance, the vertices of the graph may
+   A topological order is a linear ordering of the nodes in a graph such that
+   for every directed edge u -> v from node u to node v, node u comes
+   before node v in the ordering. For instance, the nodes of the graph may
```

Comment:
This is a substantial change that should be reviewed by the original author of the module and/or doc.

Reply:
In this context, nodes and vertices are synonyms.
The word "node" is used everywhere else in the document except for this sentence.